geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -161,17 +244,45 @@ app.MymapsDirectiveController.prototype.createMap = function() {
  * @export
  */
 app.MymapsDirectiveController.prototype.chooseMap = function() {
-  this.choosing = true;
+  if (!this.appUserManager_.isAuthenticated()) {
+    this.askToConnect();
+  }else {
+    this.appMymaps_.getMaps().then(goog.bind(function(mymaps) {
+      if (goog.isNull(mymaps)) {
+        this.askToConnect();
+      } else if (!goog.array.isEmpty(mymaps)) {
```

Comment:
If you use `goog.array.isEmpty` you need to add `goog.require('goog.array')` at the head of the file.